src/util.cpp

```diff
@@ -532,7 +532,10 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                               "<p><b>" +
                               tr ( "Slovak" ) +
                               "</b></p>"
-                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>" );
+                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>" +
+                              "<p><b>" + tr ( "Simplified Chinese" ) +
+                              "</b></p>"
```

Comment:
Hmm. This looks strange. But as far as I know, clang-format complained if you did it as above?

Reply:
> clang-format complained if you did it as above?

[Yes](https://github.com/jamulussoftware/jamulus/pull/1927#discussion_r676131598)